### ISSUE TITLE
phpcbf with PSR2 errors on control structure alternative syntax

### ISSUE THREAD

**Issue by themlkang** [squizlabs/PHP_CodeSniffer#366]
We have been running phpcbf with PSR2 standard on our php codebase and many files have caused errors.  phpcbf seemingly goes into an infinite loop until it reaches 50 passes.  We're pretty sure this is due to this syntax in our code: http://php.net/manual/en/control-structures.alternative-syntax.php
`phpcbf test.php --standard=PSR2`

```
<?php
//test.php
if ($a == 5):
    echo "a equals 5";
    echo "...";
elseif ($a == 6):
    echo "a equals 6";
    echo "!!!";
else:
    echo "a is neither 5 nor 6";
endif;
?>
```

**Comment by hackel** [comment]
@gsherwood With this fix, PHP open and close tags (e.g. in template files) get all messed up.  Is this intentional?  E.g.:
```php
<?php while($row = $data->getRow()): ?>
    <p><?= $val ?></p>
<?php endwhile; ?>
```
becomes:
```php
<?php while ($row = $data->getRow()) :
?>
    <p><?= $val ?></p>
<?php
endwhile; ?>
```
I realize PSR-2 isn't clear on this point, but I would think inline php open and close tags should be just fine for this use.

**Comment by gsherwood** [comment]
While I think that checking template files is probably not worth the effort (unless you have a specific standard for those files) I do think the sniffs should do their best not to break that code. I've committed a change to help with the code sample you've provided.

**Comment by hackel** [comment]
Brilliant!  I think ideally for templates we would need a way to combine the features of phpcs with HTML Tidy or something similar, but that seems out of the scope of this project.  The biggest violation I saw in template files was just not having a space after control structures, which this should take care of nicely.
